### ISSUE TITLE
List the number of unique hours in the dataset

### ISSUE THREAD

**Issue by dabinat** [mozilla/voice-web#2201]
The Datasets page is confusing because it makes you think you're getting 780 hours of content (in English) and then you pass it through the DeepSpeech importer script and you only get about 130 hours.

While duplicates can still be useful for some purposes and should definitely be part of the dataset, I think the Datasets page as currently written is misleading and the total number of unique hours in the set should be specified on that page.

**Comment by kdavis-mozilla** [comment]
It _is_ the number of unique hours, different people reading the same sentence do not produce the same audio.

What you want it to indicate is the number of hours in the train+dev+test set which _is_ less than the posted number of hours.

There's a question of degree here.

Some people would be happy having some sentence repetition, different people reading the same sentence, in their data set. Some people want no such repetition, as it biases results. We took most conservative approach in the train+dev+test split, no repetition of sentences. However, we also included all the validated sentences which include repetition.

So you do indeed have the advertised number of validated hours of audio, but this number of validated hours has some sentence repetition and is not, for methodological reasons, used in the default train+dev+test split. But you can create your own  train+dev+test split with the full number of advertised hours.

**Comment by dabinat** [comment]
Yes, maybe "unique hours" isn't the most accurate description - perhaps "unique sentence hours" is better. Or alternatively listing "repeated hours" instead.

I agree that repetitions are useful for some purposes and are good to have in the set. But the reason I raised this as an issue is the disparity: it's about 17% unique sentence recordings vs 83% repeats. So the dataset is considerably more useful for those that need repetitions than those that don't.

**Comment by kdavis-mozilla** [comment]
Maybe @Gregoor or @mbransn could have the wording changed?

**Comment by nukeador** [comment]
Thanks @dabinat for the suggestion, and thank you @kdavis-mozilla for outlining the methodological train+dev+test reasoning, which reflects the team’s current approach.

The DeepSpeech import is one important use case, but as both you and Kelly noted, there will be various others, for which sentence repetitions are good to be in. Thus, we would refrain from changing the current wording around validated / overall hours, which we’ve consistently used in the past.

That said, in exploration of best practices we’ll discuss internally if it makes sense to list this additional info in the description, weighing the above mentioned plus in accuracy against adding more complexity and potentially confusion. Because without this particular context, “unique sentence hours” or “repeated hours” may be confusing to other stakeholders of the project outside the ML / data science community.

Thanks!

**Comment by moz-bozden** [comment]
Closing as stale...
